### ISSUE TITLE
NTR: 5-hydroxyindoleacetic acid high in urine (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8415]
New term request 5-hydroxyindoleacetic acid high in urine (elthat_180928154105): 5-HIAA  UMLS: 4538836

**Comment by pnrobinson** [comment]
@MickeySegal
this is a relatively commonly abnormal metabolite in CSF, but I am not finding references for urine. What is the context?

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/618049
"Brain imaging and CSF neurotransmitter levels were normal, but urinary analysis showed increased levels of neurotransmitter metabolites, including homovanillic acid (HVA) and 5-hydroxyindoleacetic acid (5-HIAA), as well as decreased levels of norepinephrine and dopamine. "

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000756
- label: Elevated urinary 5-hydroxyindoleacetic acid level
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: PMID:23363473
- def: The amount of 5-hydroxyindoleacetic acid (5-HIAA) in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8415
- synonyms:
